scripts/kubestellar

```diff
@@ -240,6 +241,24 @@ EOF
     echo "Finished augmenting root:compute for KubeStellar"
 }
 
+function ensure_space_mgt() {
+    kubectl ws root &> /dev/null
+    $SCRIPT_DIR/space-provider-config "$KUBECONFIG" "$SCRIPT_DIR/../config/space/provider_kcp.yaml"
+    if kubectl get Workspace "$space_mgt_name" &> /dev/null; then
+	echo "space-mgt workspace already exists -- using it:"
+	kubectl ws "$space_mgt_name"
```

Comment:
If the space mgr already exists, do we need to set the context to it?

Reply:
Since creating the context is a separate step from creating the Workspace, yes, it is possible that one happened in the past but not the other.

Reply:
We create context in all cases. The `--overwrite` flag will overwrite the context if it's already exist.

Reply:
Ah, right.